src/UIKit/UIAdaptivePresentationControllerDelegate.cs

```diff
@@ -0,0 +1,37 @@
+#if !WATCH
+using System;
+using XamCore.UIKit;
+using XamCore.ObjCRuntime;
+using XamCore.Foundation;
+
+namespace XamCore.UIKit {
+	public partial class UIAdaptivePresentationControllerDelegate
+	{
+		// this is a workaround to allow exposing the old API ()
+		[Export ("adaptivePresentationStyleForPresentationController:")]
+		public virtual UIModalPresentationStyle GetAdaptivePresentationStyle (UIPresentationController forPresentationController)
```

Comment:
I am unsure if there is any problem with the magic that @rolfbjarne does for the protocols registration in the generator, I would double check with him first :)